brompton/sass/_config-child-theme-deep.scss

```diff
@@ -70,37 +70,37 @@ $config-global: (
 	/* Colors */
 	"color": (
```

Comment:
I believe this entire color map should be removed, and after this config is set, you merge the varia config like so: 

https://github.com/Automattic/themes/blob/f264ac103ba2b07896acb6a9887a1e88b4155160/alves/sass/_config-child-theme-deep.scss#L108